docs/index.md

```diff
@@ -35,9 +35,11 @@ that machine is also the control machine.
 - [Telling `commcare-cloud` about your environments](commcare-cloud/env)
 - [Server Management Basics](commcare-cloud/basics)
 
-## Services
+## Components
+
 - CouchDB
     - [Add a new CouchDB node to an existing cluster](howto/add-couchdb2-node.md)
+- [Kafka](components/kafka.md)
```

Comment:
@snopoke I took the liberty of handling this conflict since it was a change of mine on another PR that introduced it. We've now got `components`, `services`, and `howto` under here, so we may want to restructure these directories soon to be more consistent.

Reply:
Sorry @dannyroberts, I had actually merged this locally but forgot to push. I've moved it into `services`. In the long run I'd like to move all the 'howtos' into their related service directories as I've done with `postgresql`